doc/manual/getting_started_mgmt.rst

```diff
@@ -233,10 +233,10 @@ Then set its execution permissions: ::
 
 Let's now make a modification to the experiments. In the working directory ``artiq-work``, open ``mgmt_tutorial.py`` again and add an exclamation mark to the end of "Hello World". Before committing it, check that the experiment can still be executed correctly by submitting it directly from the working directory, using the command-line client: ::
 
-    $ artiq_client submit ~/artiq-work/mgmt_tutorial.py
+    $ artiq_client submit --content ~/artiq-work/mgmt_tutorial.py
 
 .. note::
-    Alternatively, right-click in the Explorer dock and select the 'Open file outside repository' option for the same effect.
+    The ``--content`` flag submits by content, that is, from the client's local environment, instead of from the master's. Since you are likely running the client and the master on the same machine, it is probably not strictly necessary here. You can also right-click in the Explorer dock and select the 'Open file outside repository' option for the same effect. In a distributed arrangement across multiple machines, ``--content`` must be used and the GUI option will not be available.
```

Comment:
Pretty likely, since that's what the tutorial guides you to do. `--bind` and `-s` are mentioned, but I think most people are not going to jump to get a second machine up and running before finishing the tutorial.

Reply:
That's confusing me a little. Content would in fact submit directly from artiq-work/repository, wouldn't it? I.e. as opposed to concerning itself at all with artiq-master/repository or the git data.

Reply:
Right, so the _client_ takes it directly from artiq-work, but it arrives at the _master_ as a bare file, which can be of any origins at all, so the existing phrasing is not great. I think I see.